### ISSUE TITLE
nsfminer skips the first pool parameter

### ISSUE THREAD

**Issue by oshpak** [no-fee-ethereum-mining/nsfminer#112]
nsfminer skips the first pool parameter from the list. I tested it on both Windows and Ubuntu 20.04 and the behavior is the same.

In the example below I expected that it would connect to eth-us-west.flexpool.io.

```
$ ./nsfminer -U -R --syslog --display-interval 60 --HWMON 2 --exit --nocolor --tstop 65 --tstart 50 \
> -P stratum1+tcp://${wallet}.${worker}@eth-us-west.flexpool.io:4444 \
> -P stratum1+tcp://${wallet}.${worker}@eth-de.flexpool.io:4444 \
> -P stratum1+tcp://${wallet}.${worker}@eth-us-east.flexpool.io:4444
miner nsfminer 1.3.5+commit.811f00e5 (No stinkin' fees edition)
miner Copyright 2021 Jean M. Cyr, Licensed under the terms
miner  of the GNU General Public License Version 3
miner https://github.com/no-fee-ethereum-mining/nsfminer
miner Build: linux/release/gnu
miner 3rd Party: GCC 9.3.0, CUDA 11.2, Boost 1.75.0
miner 3rd Party: OpenSSL 1.1.1i, Ethash 0.5.0
miner Configured pool eth-us-west.flexpool.io:4444
miner Configured pool eth-de.flexpool.io:4444
miner Configured pool eth-us-east.flexpool.io:4444
miner Selected pool eth-de.flexpool.io:4444
miner Stratum mode : Eth-Proxy compatible
miner Established connection to eth-de.flexpool.io:4444
miner Spinning up miners...
cu-0  Using Pci 0000:1a:00.0: TITAN RTX (Compute 7.5) Memory : 23.65 GB
cu-1  Using Pci 0000:68:00.0: TITAN RTX (Compute 7.5) Memory : 23.65 GB
miner Epoch : 397 Difficulty : 4.00 Gh
miner Job: e9a17ab5 blk: 11910390 eth-de.flexpool.io:4444
```

US-EAST is my closest location. Here is another attempt with US-EAST first:

```
$ ./nsfminer -U -R --syslog --display-interval 60 --HWMON 2 --exit --nocolor --tstop 65 --tstart 50 \
> -P stratum1+tcp://${wallet}.${worker}@eth-us-east.flexpool.io:4444 \
> -P stratum1+tcp://${wallet}.${worker}@eth-us-west.flexpool.io:4444 \
> -P stratum1+tcp://${wallet}.${worker}@eth-de.flexpool.io:4444
miner nsfminer 1.3.5+commit.811f00e5 (No stinkin' fees edition)
miner Copyright 2021 Jean M. Cyr, Licensed under the terms
miner  of the GNU General Public License Version 3
miner https://github.com/no-fee-ethereum-mining/nsfminer
miner Build: linux/release/gnu
miner 3rd Party: GCC 9.3.0, CUDA 11.2, Boost 1.75.0
miner 3rd Party: OpenSSL 1.1.1i, Ethash 0.5.0
miner Configured pool eth-us-east.flexpool.io:4444
miner Configured pool eth-us-west.flexpool.io:4444
miner Configured pool eth-de.flexpool.io:4444
miner Selected pool eth-us-west.flexpool.io:4444
miner Stratum mode : Eth-Proxy compatible
miner Established connection to eth-us-west.flexpool.io:4444
miner Spinning up miners...
cu-0  Using Pci 0000:1a:00.0: TITAN RTX (Compute 7.5) Memory : 23.65 GB
cu-1  Using Pci 0000:68:00.0: TITAN RTX (Compute 7.5) Memory : 23.65 GB
miner Epoch : 397 Difficulty : 4.00 Gh
miner Job: ee44b13f blk: 11910430 eth-us-west.flexpool.io:4444
```

I expect that miner chooses either the first parameter in the list or better yet a working pool with the shortest latency.

**Comment by oshpak** [comment]
I went back to 1.2.5 because 1.3.5 is not stable; it is causing all kinds of issues. Confirmed that the pool parameter works correctly there.

**Comment by jean-m-cyr** [comment]
Fixed by PR #114 in 1.3.6
